students/marc_charbo/lesson05/mailroom_v3.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import logging
+import logging.config
+
+#logging.cong file
+logging.config.fileConfig('logging.conf')
+# create logger
+logger = logging.getLogger('mailRoom')
+
+DONOR_LIST = {'Jim':[25.00,150.00,2000.00,100000.00],'Linda':[10000.25],'Bob':[5.03,100.01,6.00]}
+
+def send_letter():
+    """write thank you note to all users in donor list"""
+    for key, value in DONOR_LIST.items():
+        file = open(key+'.txt', 'w')
```

Comment:
try using a `with` context manager here

Reply:
@mdgregor could you elaborate on using with? thanks